tests/UrlTest.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+/*
+ * This file is part of the webmozart/path-util package.
+ *
+ * (c) Bernhard Schussek <bschussek@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Webmozart\PathUtil\Tests;
+
+use Webmozart\PathUtil\Url;
+
+/**
+ * @since  2.2
+ *
+ * @author Bernhard Schussek <bschussek@gmail.com>
+ * @author Claudio Zizza <claudio@budgegeria.de>
+ */
+class UrlTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @dataProvider provideMakeRelativeTests
+     * @covers Webmozart\PathUtil\Url
+     */
+    public function testMakeRelative($absoluteUrl, $baseUrl, $relativePath)
+    {
+        $relative = Url::makeRelative($absoluteUrl, $baseUrl);
+        $this->assertSame($relativePath, $relative);
+    }
+
+    /**
+     * @dataProvider provideMakeRelativeTests
+     * @covers Webmozart\PathUtil\Url
+     */
+    public function testMakeRelativeWithUrl($absoluteUrl, $baseUrl, $relativePath)
+    {
+        $relative = Url::makeRelative($this->createUrl($absoluteUrl), $this->createUrl($baseUrl));
```

Comment:
No, the data provider can stay the same. If you check my code sample, it really just replaces the `createUrl()` call. :)